diff --git a/lisp/gnus/gnus-icalendar.el b/lisp/gnus/gnus-icalendar.el
index ee556a3..5ac1d78 100644
--- a/lisp/gnus/gnus-icalendar.el
+++ b/lisp/gnus/gnus-icalendar.el
@@ -358,8 +358,8 @@ on the IDENTITIES list."
 ;; TODO: this is an optional feature, and it's only available with org-mode
 ;; 7+, so will need to properly handle emacsen with no/outdated org-mode
 
-(require 'org)
-(require 'org-capture)
+(require 'org nil t)
+(require 'org-capture nil t)
 
 (defgroup gnus-icalendar-org nil
   "Settings for Calendar Event gnus/org integration."
